vividus-bdd-engine/src/main/java/org/vividus/bdd/steps/SubSteps.java

```diff
@@ -88,6 +88,10 @@ else if (rootCause instanceof Error)
         {
             throw (Error) rootCause;
         }
+        else if (rootCause instanceof UUIDExceptionWrapper)
+        {
+            throw new UUIDExceptionWrapper(rootCause);
```

Comment:
double-wrapping?

Reply:
- https://github.com/jbehave/jbehave-core/blob/master/jbehave-core/src/main/java/org/jbehave/core/steps/StepCreator.java#L889
- and as per unit tests
- technically this is a single wrapping
<img width="184" alt="Screenshot 2021-05-19 at 11 59 14" src="https://user-images.githubusercontent.com/30923336/118785655-b91b1900-b899-11eb-92af-c3b020db38fb.png">

Reply:
`throw stepResult.getFailure();` ?